composer.json

```diff
@@ -55,28 +56,28 @@
         "sonata-project/admin-bundle": "<3.59",
         "sonata-project/block-bundle": "<3.18",
         "sonata-project/core-bundle": "<3.20",
-        "sonata-project/doctrine-orm-admin-bundle": "<3.0 || >=4.0",
-        "sonata-project/notification-bundle": "<3.0 || >=4.0",
-        "sonata-project/seo-bundle": "<2.0 || >=3.0"
+        "sonata-project/doctrine-orm-admin-bundle": "<3.5 || >=4.0",
+        "sonata-project/notification-bundle": "<3.4 || >=4.0",
+        "sonata-project/seo-bundle": "<2.5 || >=3.0"
     },
     "require-dev": {
         "doctrine/orm": "^2.4",
-        "friendsofsymfony/rest-bundle": "^1.8 || ^2.1",
-        "jms/serializer-bundle": "^2.0 || ^3.0",
+        "friendsofsymfony/rest-bundle": "^2.3",
+        "jms/serializer-bundle": "^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1",
-        "nelmio/api-doc-bundle": "^2.4",
+        "nelmio/api-doc-bundle": "^2.13.3",
         "sonata-project/admin-bundle": "^3.59",
         "sonata-project/block-bundle": "^3.18",
-        "sonata-project/doctrine-orm-admin-bundle": "^3.4",
-        "sonata-project/notification-bundle": "^3.3",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.5",
+        "sonata-project/notification-bundle": "^3.4",
         "sonata-project/seo-bundle": "^2.5",
-        "symfony/phpunit-bridge": "^5.0"
+        "symfony/phpunit-bridge": "^5.1"
     },
     "suggest": {
         "nelmio/api-doc-bundle": "If you want to use the API",
         "sonata-project/block-bundle": "For rendering block lists.",
-        "sonata-project/doctrine-orm-admin-bundle": "^2.2",
-        "sonata-project/notification-bundle": "^2.2"
+        "sonata-project/doctrine-orm-admin-bundle": "^3.5",
```

Comment:
this is not for the version, can you set a message suggesting why install the orm admin bundle? same for the next line

Reply:
Done